flang/lib/Optimizer/Builder/IntrinsicCall.cpp

```diff
@@ -3230,6 +3235,22 @@ void IntrinsicLibrary::genExecuteCommandLine(
                                       exitstatBox, cmdstatBox, cmdmsgBox);
 }
 
+// ETIME
+void IntrinsicLibrary::genEtime(
+    llvm::ArrayRef<fir::ExtendedValue> args) {
+  assert(args.size() == 2);
+
+  mlir::Value values = fir::getBase(args[0]);
+  mlir::Value time = fir::getBase(args[1]);
+
+  if (!values)
+    fir::emitFatalError(loc, "expected VALUES parameter");
```

Comment:
Fortran uses "argument" instead of "parameter".

Reply:
Ok, but I was referring to the code here. So, should this also be "arguments"?
https://github.com/llvm/llvm-project/blob/410d6350eda322de213941b36adcdab13a0b557b/flang/lib/Optimizer/Builder/IntrinsicCall.cpp#L3193-L3194
https://github.com/llvm/llvm-project/blob/410d6350eda322de213941b36adcdab13a0b557b/flang/lib/Optimizer/Builder/IntrinsicCall.cpp#L3425-L3426

Reply:
Yes, I think that needs to be changed as well. Perhaps we can do it in a separate PR.

Reply:
Yes, let's keep this PR focused on the `etime` intrinsic.  I will make a PR to clean this up (and also fix up the `elemental` issue with most of the intrinsics, as @klausler as noted correctly).